technicalreport/stateoftheart.tex

```diff
@@ -118,24 +118,305 @@ \subsubsection{Schweizer Norm 640 290}
+        \bottomrule
+    \caption{Ermittlung der Haltestellenkategorie Kanton Zürich}
+    \label{table:Ermittlung der Haltestellenkategorie Kanton Zürich}
+\end{longtable}
+
+
+
+
+\subparagraph{Erreichbarkeit der Haltestelle}~\\
+\label{Berechnungsmethodik Kanton Zürich:Erreichbarkeit der Haltestelle}
+Auch im Kanton Zürich werden durch den zusätzlich abgebildeten Kursintervall (> 60 Minuten) zwei neue Güteklassen E und F eingeführt.
+
+\begin{longtable}[c]{l p{3.3cm} p{3.3cm} p{3.3cm} p{3.3cm}}
+        \midrule
+        \textbf{}
+                                & \textbf{< 300 m}
+                                & \textbf{300 - 500 m}
+                                & \textbf{501 - 750 m}
+                                & \textbf{751 - 1000m}\\
+        \textbf{I}
+                                & Klasse A
+                                & Klasse A
+                                & Klasse B
+                                & Klasse C\\
+        \textbf{II}
+                                & Klasse A
+                                & Klasse B
+                                & Klasse C
+                                & Klasse D\\
+        \textbf{III}
+                                & Klasse B
+                                & Klasse C
+                                & Klasse D
+                                &\\
+        \textbf{IV}
+                                & Klasse C
+                                & Klasse D
+                                & \cellcolor{red!25}Klasse E
+                                &\\
+        \textbf{V}
+                                & Klasse D
+                                & \cellcolor{red!25}Klasse E
+                                & \cellcolor{red!25}Klasse E
+                                &\\
+        \cellcolor{red!25}\textbf{VI}
+                                & \cellcolor{red!25}Klasse E
+                                & \cellcolor{red!25}Klasse E
+                                &
+                                &\\
+        \cellcolor{red!25}\textbf{VII}
+                                & \cellcolor{red!25}Klasse F
+                                & \cellcolor{red!25}Klasse F
+                                &
+                                &\\                                
+        \bottomrule
+    \caption{Ermittlung der Erreichbarkeit der Haltestelle Kanton Zürich}
+    \label{table:Ermittlung Erreichbarkeit der Haltestelle Kanton Zürich}
+\end{longtable}
+
+Der Topografie wird keine Rechnung getragen, es wird strikt auf die Luftlinie gesetzt.
+Sind Haltestellen nur durch Tram und/oder Bus erschlossen, werden analog zum Kanton Graubünden nur Distanzen bis zu 500 Meter berücksichtigt.
+Die Erschliessung durch Bahnknoten und Bahnlinien wird bis 750 Meter mit den Güteklassen B – E und bis 1000 Meter mit den Güteklassen C-D berücksichtigt.
+Diese Entscheidungen basieren auf ÖV-Angebotsverordnungen.
+
 \paragraph{Problematik}~\\
```

Comment:
Dieser Paragraph pass hier irgendwie nicht mehr rein. Soll es ein eigenes Unterkapitel geben, wo wir die einzelnen Probleme ansprechen?
Teile davon sind auch etwas redunant, da die Probleme der Norm ja schon in vielen Punkten von ARE / GB / ZH gelöst wurden

Reply:
Dies wird mit Issue 97 überarbeitet werden, da ich auch noch die Ziele und Unterziele aufräumen möchte und dies in einen Spezifikationsteil schieben will.